plugins/woocommerce-blocks/assets/js/blocks/checkout/inner-blocks/checkout-shipping-methods-block/block.tsx

```diff
@@ -60,7 +60,11 @@ const renderShippingRatesControlOption = (
 	};
 };
 
-const Block = ( { noShippingPlaceholder = null } ): ReactElement | null => {
+const Block = ( {
+	noShippingPlaceholder = null,
+}: {
+	noShippingPlaceholder?: ReactElement | null;
+} ): ReactElement | null => {
```

Comment:
I realize you didn't introduce this, but you don't need to explicitly state return types, TS will figure this out, so it can be removed.

Reply:
https://github.com/woocommerce/woocommerce/pull/52192/commits/adfef87b1e2aec6fb58710422ab44dd0df9f5fdb